WMCDuplicateRemover/Code/Wrappers/MicrosoftEventLogWrapper.cs

```diff
@@ -45,14 +46,12 @@ private void CreateFileCache(HashSet<string> eventLogCache)
         public override bool FoundEventForRecording(string seriesName, string episodeName)
         {
             //Can't be cancelling shows we only have part of the information needed!
-            if (string.IsNullOrEmpty(seriesName) || string.IsNullOrEmpty(episodeName))
+            if (string.IsNullOrWhiteSpace(seriesName) || string.IsNullOrWhiteSpace(episodeName))
                 return false;
 
-            //These are exceptions since I had to go and delete off these because WMC screwed up its DRM and I wasn't actually
-            //finished watching these
-            //TODO: Add the ability to add exceptions to this program without hard coding here
-            //if (seriesName.ToLower() == "True Detective".ToLower() || seriesName.ToLower() == "Silicon Valley".ToLower())
-            //    return false;
+            var exlusions = UserSettings.Default.Exclusions.Cast<string>().Select(x => x.ToLower());
```

Comment:
It's called the lambda operator. Check out this link and let me know if you have any questions. https://msdn.microsoft.com/en-us/library/bb311046.aspx